wcfsetup/install/files/lib/system/user/notification/event/ModerationQueueCommentResponseUserNotificationEvent.class.php

```diff
@@ -234,4 +216,27 @@ protected static function getTestCommentObjectData(UserProfile $recipient, UserP
             'objectTypeID' => CommentHandler::getInstance()->getObjectTypeID('com.woltlab.wcf.moderation.queue'),
         ];
     }
+
+    /**
+     * @inheritDoc
+     */
+    protected function getTypeName(): string
+    {
+        if ($this->typeName === null) {
```

Comment:
```suggestion
        if (!isset($this->typeName)) {
```
`isset()` is required to work with typed properties that might be *uninitialized*.